js/TextareaEvents.js

```diff
@@ -0,0 +1,16 @@
+
+function textareaEvents(id) {
+    let field = document.getElementById(id);
+    field.onfocus = function () {
+        if (this.value == this.defaultValue) {
+            this.value = '';
+        }
+    };
+    field.onblur = function () {
+        if (this.value == '') {
+            this.value = this.defaultValue;
```

Comment:
Why do you need default value? Why not just use placeholder?

Reply:
what is it placeholder?

Reply:
> what is it placeholder?

It is `textarea` attribute. Placeholder text will be displayed when there is no value. So I guess you don't need that function